### ISSUE TITLE
Support for ID3v2.4

### ISSUE THREAD

**Issue by TheyCallMeHacked** [MuntashirAkon/Metro#105]
**Is your feature request related to a problem? Please describe.**

I have a few songs that have multiple artists. On all of these–they use ID3v2.4 instead of ID3v2.3–only the first one is ever shown

**Describe the solution you'd like**

I'd like the App to support ID3v2.4 tags (where multiple values are separated by a NULL byte instead of a slash) 

**Describe alternatives you've considered**

I've tried to manually convert the tags to ID3v2.3 (basically changing the version bytes and replacing the NULL byte in question by a `/`) and it does "work"; it shows both artists separated by a slash, not formatted to say "and" or something...

**Additional context**

What I get with ID3v2.4
![Screenshot_20220629-104333.png](https://user-images.githubusercontent.com/57837439/176394088-7ccd630f-805a-4871-8f15-10634d092b0f.png)

What I get with ID3v2.3
![Screenshot_20220629-104322.png](https://user-images.githubusercontent.com/57837439/176394258-734c110b-d8f4-474e-989a-6f804510287d.png)

What I would have expected (with both ID3v2.3 and ID3v2.4)
![Screenshot_20220629-104333~2.png](https://user-images.githubusercontent.com/57837439/176395597-16d5d33e-bebb-4ffd-8fe6-1db4250d098b.png)

**Comment by Teraskull** [comment]
Yes, a dot divider would be nice, like how the artist and album are divided. Tagging artists in mp3tag using "artist1\\artist2" creates 2 artist fields, and Metro only shows the first field.
